x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/select_connector/select_connector.tsx

```diff
@@ -77,20 +91,69 @@ export const SelectConnector: React.FC = () => {
         <EuiFormFieldset
           legend={{
             children: (
-              <EuiText color="subdued" size="s">
-                <p />
-              </EuiText>
+              <EuiCallOut
+                size="m"
+                title={i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.selectConnector.callout.title',
+                  { defaultMessage: 'Elastic connectors' }
+                )}
+                iconType="iInCircle"
+              >
+                <p>
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.content.indices.selectConnector.description.textcloud"
+                    defaultMessage="Use {native} connectors out of the box on Elastic cloud. {learnMore}"
```

Comment:
Text should read:


Native connectors are available directly within Elastic Cloud deployments. No additional infrastructure is required. Self-managed deployments must deploy the connector service to run native connectors. {learnMore}

Let me know if that looks OK :)

Reply:
![Screenshot 2023-05-10 at 19 51 06](https://github.com/elastic/kibana/assets/1410658/31861778-aa34-4c6d-81ed-db6fe69fe15f)

Does it look good this way?

Reply:
Just pushed the changes, let me know if you need any other changes <3